styles/globals.css

```diff
@@ -75,4 +75,27 @@
   body {
     @apply bg-background text-foreground;
   }
+}
+
+/* Scrollbar styles*/
+/* width */
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px grey; 
+  border-radius: 10px;
+}
+ 
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #888; 
+  border-radius: 10px;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #555; 
 }
```

Comment:
do you want this scrollbar effect on the entire website? or only on some specific components? 
If you are trying to just use this effect for some components/pages then, creating a custom utility class would be better I think. 
reference: [here](https://v1.tailwindcss.com/docs/plugins#adding-utilities)

Reply:
This was intentional. It looks nicer than default one. Though I can change that if needed

Reply:
Could you please provide a visual as I haven't run this PR locally yet.

Reply:
https://github.com/mfts/papermark/assets/73285842/c5898af6-19f7-4016-b463-3c3acfaff5f0